taskcluster/scripts/bitrise-schedule.py

```diff
@@ -50,7 +50,7 @@ def sync_main(
     parser.add_argument("--branch", required=True, help="the git branch to generate screenshots from")
     parser.add_argument("--commit", required=True, help="the git commit hash to generate screenshots from")
     parser.add_argument("--workflow", required=True, help="the bitrise workflow to schedule")
-    parser.add_argument("--locale", required=True, help="locale to generate the screenshots for")
+    parser.add_argument("--locale", dest="locales", metavar="LOCALE", action="append", required=True, help="locale to generate the screenshots for (can be repeated)")
```

Comment:
This change enables the script to be called this way: `bitrise-schedule.py --locale en-CA --locale es --locale fr`